content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -545,15 +554,17 @@ work between Windows and Linux:
 
 The following list documents differences between how Pod specifications work between Windows and Linux:
 
-* `hostIPC` and `hostpid` - host namespace sharing is not possible on Windows
+* `hostIPC` and `hostpid` - host namespace sharing is not possible on Windows. Pod API
+   validation would fail when `IdentifyPodOS` featuregate is enabled.   
 * `hostNetwork` - There is no Windows OS support to share the host network
 * `dnsPolicy` - setting the Pod `dnsPolicy` to `ClusterFirstWithHostNet` is
    not supported on Windows because host networking is not provided. Pods always
    run with a container network.
 * `podSecurityContext` (see below)
 * `shareProcessNamespace` - this is a beta feature, and depends on Linux namespaces
   which are not implemented on Windows. Windows cannot share process namespaces or
-  the container's root filesystem. Only the network can be shared.
+  the container's root filesystem. Only the network can be shared. Pod API
```

Comment:
side note, with HPC we do share the root filesystem not sure if that is worth noting since it is out of context of shared namespaces